### PR TITLE
Please add this a great king story, @casey-collab

### PR DESCRIPTION
**Pull request by dojha00**
Please add this a great king story, @casey-collab

### CODE REVIEW

File: hindi/bollywood.md [hunk 1]
```diff
@@ -26,3 +26,5 @@ Art of Living [Know More](art-of-living/art-of-living.md)
 [Sunny Deol](Sunny_Deol/sunny.md)
 
 [चुटकुला पढो](joke/joke.md) और मुस्कुराते रहो
+
+Click [Here](king/king.md) to read the story of a great King.
```
Comment (casey-collab) [inline hindi/bollywood.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: hindi/king.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Once upon a time there was a great king in the far away galaxy called "Swarg".
```
Comment (casey-collab) [inline hindi/king.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: hindi/king.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+And then one day he came to another galaxy called "Milky Way" on a vocation.
+After long wondering he found earth...
+and now days he is busy in writing stories on GitHub.
+To read more about him click [Here](www.google.com) and search for him.
```
Comment (casey-collab) [inline hindi/king.md]: This link is also invalid.

File: hindi/king.md (hunk 2 above)
Comment (casey-collab) [inline hindi/king.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by dojha00** [comment]
Please review this. @casey-collab

File: hindi/king/king.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Once upon a time there was a great king in the far away galaxy called "Swarg".
+And then one day he came to another galaxy called "Milky Way" on a vocation.
+After long wondering he found earth...
+and now days he is busy in writing stories on GitHub.
+To read more about him click [Here](www.google.com) and search for him.
```
Comment (casey-collab) [inline hindi/king/king.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by dojha00** [comment]
Please review once more fixed broken links. @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ice_cream:

**Pull request merged**